css/components/badge.css

```diff
@@ -71,3 +71,14 @@
   height: 1.5rem;
   min-width: 1.5rem;
 }
+
+.badge-pill {
+  background-color: var(--clr-primary);
+  color: white;
```

Comment:
suggestion : you can use variables for color

Reply:
Done. Thanks for your suggestion!